src/core.c/Exception.pm6

```diff
@@ -3184,10 +3184,11 @@ my class X::MultipleTypeSmiley does X::Comp {
 }
 
 my class X::Seq::Consumed is Exception {
+    has Str:D $.kind = "Seq";
     method message() {
-        "The iterator of this Seq is already in use/consumed by another Seq\n" ~
-        "(you might solve this by adding .cache on usages of the Seq, or\n" ~
-        "by assigning the Seq into an array)"
+        ("The iterator of this $!kind is already in use/consumed by another $!kind" ~
+        " (you might solve this by adding .cache on usages of the $!kind, or " ~
+        "by assigning the $!kind into an array)").naive-word-wrapper
```

Comment:
Hm what is `naive-word-wrapper`? I'd expect such things to be applied at the point of exception printing, not in the message production...

Reply:
But we currently don't do it anyway. There are a couple other exceptions doing this. This particular message was previously using newlines inside the message to wrap it. So, I'd keep it this way until the printing starts wrapping.

Reply:
Yes, to be fair this was a more general grumble rather than anything that should block this PR.